### PR TITLE
Adds placeholder app

### PR DESCRIPTION
**Pull request by lomky**
🦡🦡🦡🍄🍄🐍

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -5,59 +5,28 @@ export default function Home() {
   return (
     <div className={styles.container}>
       <Head>
-        <title>Create Next App</title>
+        <title>Placeholder Claimtracker App</title>
```
Comment (kalvinwang) [inline pages/index.js:8]:
```suggestion
        <title>Placeholder Claim Tracker App</title>
```

File: pages/index.js [hunk 2]
```diff
@@ -5,59 +5,28 @@ export default function Home() {
...
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
       <main className={styles.main}>
         <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
+          Welcome to the Placeholder Claimtracker App
```
Comment (kalvinwang) [inline pages/index.js:14]:
```suggestion
          Welcome to the Placeholder Claim Tracker App
```

File: pages/index.js [hunk 3]
```diff
@@ -5,59 +5,28 @@ export default function Home() {
...
       <footer className={styles.footer}>
         <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+          href="https://navapbc.com"
           target="_blank"
           rel="noopener noreferrer"
         >
-          Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
+          Powered by NavaPBC
```
Comment (kalvinwang) [inline pages/index.js:29]:
```suggestion
          Powered by Nava PBC
```

**Review by kalvinwang — APPROVED** [review]
extremely minor nits-- gotta stay on (content) brand! :P

File: pages/index.js [hunk 4]
```diff
@@ -5,59 +5,28 @@ export default function Home() {
...
+        <div>
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/EIyixC9NsLI" title="YouTube video player" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
         </div>
+
       </main>
 
       <footer className={styles.footer}>
         <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+          href="https://navapbc.com"
```
Comment (lomky) [inline pages/index.js:25]:
```suggestion
          href="https://www.navapbc.com/"
```

**Pull request merged**